ide/editor.codetemplates/src/org/netbeans/lib/editor/codetemplates/CodeTemplateInsertHandler.java

```diff
@@ -109,7 +109,7 @@ public final class CodeTemplateInsertHandler implements TextRegionManagerListene
     private TextSyncGroup textSyncGroup;
     
     private boolean completionInvoked;
-    
+
```

Comment:
I simply marked the first unrelated change in the PR and added the comment, that is why a whitespace change is selected :)

Personally, I prefer more commits (the cleanup one can be the first one in PR so no need to rebase/resolve conflicts :) because even in this simple one, I kind-of had to search what changed exactly :)

Just my nitpicks, nothing serious, of course.